admin/class-smaily-for-wp-admin.php

```diff
@@ -383,16 +309,48 @@ public function smaily_admin_save() {
 			$data       = $wpdb->get_row( "SELECT * FROM `$table_name` LIMIT 1" );
 			$template->assign( (array) $data );
 
-			// Load autoresponders.
-			$table_name = esc_sql( $wpdb->prefix . 'smaily_autoresponders' );
-			$data       = $wpdb->get_results( "SELECT * FROM `$table_name`" );
-			$template->assign( 'autoresponders', $data );
-
 			// Render template.
 			$result['content'] = $template->render();
 		}
 		// Display result messages as JSON.
 		echo json_encode( $result );
 		wp_die();
 	}
+
+	/**
+	 * Make a request to Smaily asking for autoresponders.
+	 * Request is authenticated via saved credentials.
+	 *
+	 * @return array $autoresponder_list List of autoresponders in format [id => title].
+	 */
+	public function get_autoresponders() {
+		global $wpdb;
+
+		// Load configuration data.
+		$table_name      = esc_sql( $wpdb->prefix . 'smaily_config' );
+		$config          = (array) $wpdb->get_row( "SELECT * FROM `$table_name` LIMIT 1" );
```

Comment:
Shouldn't the formatting here be more like:
```suggestion
		$table_name = esc_sql( $wpdb->prefix . 'smaily_config' );
		$config     = (array) $wpdb->get_row( "SELECT * FROM `$table_name` LIMIT 1" );
```

Reply:
Added 81619ab